tests/cases/standalone/common/system/information_schema.result

```diff
@@ -31,25 +31,43 @@ where table_catalog = 'greptime'
   and table_schema != 'public'
 order by table_schema, table_name;
 
-+---------------+--------------+------------+------------+--------+
-| table_catalog | table_schema | table_name | table_type | engine |
-+---------------+--------------+------------+------------+--------+
-| greptime      | my_db        | foo        | BASE TABLE | mito   |
-+---------------+--------------+------------+------------+--------+
++---------------+--------------------+------------+-----------------+--------+
+| table_catalog | table_schema       | table_name | table_type      | engine |
++---------------+--------------------+------------+-----------------+--------+
+| greptime      | information_schema | columns    | LOCAL TEMPORARY |        |
+| greptime      | information_schema | tables     | LOCAL TEMPORARY |        |
+| greptime      | my_db              | foo        | BASE TABLE      | mito   |
++---------------+--------------------+------------+-----------------+--------+
```

Comment:
Considering the purpose of the test, just add a filter condition `table_schema != 'information_schema'`
```suggestion
  and table_schema != 'public'
  and table_schema != 'information_schema'
order by table_schema, table_name;

| table_catalog | table_schema | table_name | table_type | engine |
+---------------+--------------+------------+------------+--------+
| greptime      | my_db        | foo        | BASE TABLE | mito   |
+---------------+--------------+------------+------------+--------+
```